### DISCUSSION TITLE
What are some tips to reduce database egress?

### DISCUSSION THREAD

**Discussion by 72L** [supabase/supabase#14867] [Questions]
What are some tips to reduce database egress? It's the only metric for me that is at risk of exceeding quota. 

How can I determine which api requests are resulting in the most egress?

When I POST data to the data base (i.e. insert) it seems to count as egress. Why? Is there something I'm doing wrong?

**Comment by GaryAustin1** [comment]
The biggest thing is obviously only fetch the data you need at the time with just certain columns versus the entire row and doing pagination when you can.   

As far as insert, if you have .select() on the end then you should only return the data columns you need (like the primary id).  Note that my understanding is that the return status and headers and such also count in egress.
